spec/index.html

```diff
@@ -1467,44 +1467,53 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
 
       <dt id="JSON-value-space">The <a>value space</a></dt>
       <dd>
-        is a single <a data-cite="RFC4627#section-2.1">JSON value</a> in the form of an <a data-cite="INFRA#list">array</a>,
-        <a data-cite="INFRA#ordered-map">map</a>,
-        <a data-cite="INFRA#string">string</a>,
-        <a data-cite="ECMASCRIPT#sec-terms-and-definitions-number-value">number</a>,
-        <a data-cite="INFRA#boolean">boolean</a>, or
-        <a data-cite="INFRA#nulls">null</a>.
+        is the set of all <a data-lt="string">RDF strings</a> that conform to the
+        <a data-cite="RFC4627#section-2">JSON Grammar</a> as described in
+        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
+        [[RFC8259]], and furthermore comply with the following constraints:
         <ul>
-          <li><a data-cite="INFRA#list">Array</a> <a data-cite="INFRA#list-item">entries</a> may be
-            any of the above <a data-cite="RFC4627#section-2.1">JSON values</a>.</li>
-          <li><a data-cite="INFRA#ordered-map">Map</a> <a data-cite="INFRA#map-key">keys</a> are <a data-cite="INFRA#string">strings</a> with
-            <a data-cite="INFRA#map-value">values</a>, which may be any of the above
-            <a data-cite="RFC4627#section-2.1">JSON values</a>.</li>
+          <li>They MUST NOT contain any unnecessary whitespace.</li>
+          <li>Keys in objects MUST be ordered lexicographically.</li>
```

Comment:
uh-oh. `lexicographically`, again?

Reply:
The section can still be improved, but I was trying to just go back to the JSON-LD 1.1 definition. We can either improve that, or go in the native value-space suggested previously and in #65. At this point, the fewer moving parts the better.

Reply:
Fewer moving parts is generally better, yes.

Still, we now know that `lexicographical` ordering is ambiguous at best, so I think we should replace that with the ordering we believe to be most unambiguous and closest to the original intent, which I *think* now means `Unicode code point order` (or something close to that).